auth/auth-router.js

```diff
@@ -50,6 +50,8 @@ router.post("/login", (req, res) => {
                 res.status(200).json({
                     message: `Welcome Back!`,
                     token,
+                    userType:`${user.user_type}`,
```

Comment:
Thanks for making this change, it will make things a lot easier on the front end